Src/library/net/Clib/network.c

```diff
@@ -775,11 +778,108 @@ EIF_INTEGER c_put_stream_noexception(EIF_INTEGER fd, EIF_POINTER stream_pointer,
 }
 
 void c_put_stream(EIF_INTEGER fd, EIF_POINTER stream_pointer, EIF_INTEGER length)
-	/*x transmission of string s of size size trought socket fd */
+	/*x transmission of string s of size length throught socket fd */
 {
 	eif_net_check(c_put_stream_noexception(fd, stream_pointer, length));
 }
 
+EIF_INTEGER c_unoptimized_sendfile_noexception(EIF_INTEGER out_fd, FILE* f, EIF_INTEGER length)
```

Comment:
We should reject a length of 0 by default. If the file needs to be calculated, we should do that in the Eiffel code. This will avoid the need of refering `eif_file.h' which is runtime specific for EiffelBase and should not be used anywhere else.